src/docfx/lib/HtmlUtility.cs

```diff
@@ -94,6 +95,13 @@ internal static class HtmlUtility
             { "iframe", new HashSet<string>(StringComparer.OrdinalIgnoreCase) { "allowfullscreen", "height", "src", "width" } },
         };
 
+        private static readonly List<string> s_cjkCharUnicodeRangeList = new List<string>
+        {
+            "^[\u2E80-\u9FFF]$",    // CJK character
```

Comment:
This regex basically means any character whose char code is between `\u2E80` and `\u9FFF`, so this is equivalent to `c >= '\u2E80' && c <= '\u9FFF'`